test/trento/application/integration/checks/wanda/wanda_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Trento.Integration.Checks.WandaTest do
+  @moduledoc false
+
+  use ExUnit.Case
+
+  import Mox
+
+  alias Trento.Integration.Checks.Wanda
+
+  alias Trento.Checks.V1.ExecutionRequested
+
+  describe "wanda adapter" do
+    test "should request an execution properly" do
+      execution_id = UUID.uuid4()
+      group_id = UUID.uuid4()
+      provider = :some_provider
+      selected_checks = ["check_1", "check_2"]
+
+      hosts = [
+        %{host_id: "agent_1"},
+        %{host_id: "agent_2"}
+      ]
+
+      expect(Trento.Messaging.Adapters.Mock, :publish, fn topic, message ->
+        assert "executions" = topic
+        assert %ExecutionRequested{execution_id: ^execution_id} = message
```

Comment:
We could add a couple of assertions to check that the correct targets and checks selections are involved.
And while writing I realize that those have been considered in the `MapperTest`.
Ignore this :sweat_smile: